CPAC/pipeline/utils.py

```diff
@@ -20,10 +20,41 @@
 
 from CPAC.func_preproc.func_motion import motion_estimate_filter
 from CPAC.utils.bids_utils import insert_entity
+from nipype import Node, Workflow, Function
+from CPAC.pipeline import nipype_pipeline_engine as pe
+from nipype.interfaces import afni
+from nipype.interfaces.afni import Info
+import os
+import pandas as pd
 
 MOVEMENT_FILTER_KEYS = motion_estimate_filter.outputs
 
 
+def find_orientation(input_file):
+    import subprocess
+    cmd_3dinfo = [
+        "3dinfo",
+        "-orient", input_file
+    ]
```

Comment:
```suggestion
    cmd_3dinfo = ["3dinfo", "-orient", input_file]
```

[pre-commit](https://github.com/FCP-INDI/C-PAC/blob/dbdacd1c6a745b4847b2c6abdf2c2f3d6150511f/CONTRIBUTING.md#pre-commit) should have caught this?